README.md

```diff
@@ -48,12 +48,18 @@ Note that the access key and secret key provided here is authorized to perform _
 
 ### Step 3: Create MinIO Tenant
 
-We can create the Tenant now. Before that, please ensure you have requisite nodes and drives in place and relevant PVs are created. In below example, we ask MinIO Operator to create a Tenant with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVCs of 1 Ti each, per node, and total of 4 nodes, before attempting to create the MinIO Tenant.
+We can create the Tenant now. Before that, please ensure you have requisite nodes and drives in place and relevant PVs are created. In below example, we ask MinIO Operator to create a Tenant yaml with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVCs of 1 Ti each, per node, and total of 4 nodes, before attempting to create the MinIO Tenant.
 
 We recommend [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md) to create PVs.
 
 ```sh
-kubectl minio tenant create --name tenant1 --secret tenant1-secret --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns --storage-class direct.csi.min.io
+kubectl minio tenant create-yaml --name tenant1 --secret tenant1-secret --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns --storage-class direct.csi.min.io > tenant.yaml
+```
+
+Once you verify and optionally add any other relevant fields to the file `tenant.yaml` created by the operator, create the tenant
```

Comment:
I wonder if we should be prescriptive here.

a) Does the YAML file work as-is without modification? If so we should state "You can use the YAML file as-is to create a MinIO tenant based on the specified configuration. 

b) Prescriptive in the sense that if we know there are a few common fields to modify, maybe we should mention them. SImilarly,if there are fields users should *not* modify, we should also mention those.